.gitignore

```diff
@@ -1,7 +1,5 @@
 *.csv
```

Comment:
is it work prefixing with `**/` to exclude from all folders?

Reply:
Tested this, that is not necessary, it already excludes them deep into the folder structure.